src/posix/README.md

```diff
@@ -96,7 +96,7 @@ sudo wpantund -s 'system:./output/posix/x86_64-unknown-linux-gnu/bin/ot-ncp ./ou
 
 ```sh
 # nRF52840
-sudo wpantund -s 'system:./output/posix/x86_64-unknown-linux-gnu/bin/ot-ncp /dev/ttyACM0 115200'
+sudo wpantund -s 'system:./output/posix/x86_64-unknown-linux-gnu/bin/ot-ncp /dev/ttyACM0 115200N1H'
```

Comment:
this may not be supported. need check wpantund's configuration file.

Reply:
it seems the arguments are passed to `ot-ncp` rather than wpantund, right?

Reply:
Oh yes, I was confused.